AutoFinance.Broker/InteractiveBrokers/EventArgs/ErrorEventArgs.cs

```diff
@@ -46,5 +47,6 @@ public string ErrorMessage
             get;
             private set;
         }
+        public string AdvancedOrderRejectJson { get; }
```

Comment:
please follow the formatting pattern

```suggestion
        public string AdvancedOrderRejectJson 
        { 
            get; 
            private set;
        }
```